Library/ViewModels/ChangeEmailViewModelTests.swift

```diff
@@ -55,9 +55,7 @@ final class ChangeEmailViewModelTests: TestCase {
   }
 
   func testChangeEmail_OnSuccess() {
-    let updatedEmail = UserEmailFields.template |> \.email .~ "apple@kickstarter.com"
-    let response = UserEnvelope<UserEmailFields>(me: updatedEmail)
```

Comment:
`response` wasn't being used in the `MockService`. It wasn't being checked anywhere.